openarc/openarc.c

```diff
@@ -1768,16 +1768,18 @@ arcf_config_setlib(struct arcf_config *conf, char **err)
 	{
 		char *s = conf->conf_signhdrs_raw;
 		int cnt;
-		for (cnt=0; s[cnt]; s[cnt]==',' ? cnt++ : *s++);
+		for (cnt=0;
+		     s[cnt];
+		     s[cnt]==',' ? cnt++ : *s++);
```

Comment:
Should be immediately followed by "continue", rather than having that be implicit.